cluster-up/cluster/kind-1.27-sriov/README.md

```diff
@@ -67,6 +67,11 @@ export KUBECTL_PATH="/usr/bin/kubectl"
 This allows users to test or use custom images / different kind versions before making them official.
 See https://github.com/kubernetes-sigs/kind/releases for details about node images according to the kind version.
 
+### Running kind without SR-IOV
```

Comment:
Since this is an SR-IOV provider, the way I see it, there is no sense in running it without all the configurations it does for SR-IOV.
If you want to allow spinning up basic KIND cluster for kubevirt, I suggest introducing some kind of basic provider to allow that.
One that have all basic configurations necessary for kubevirt (e.g.: registry).

It doesn't have to be a formal provider, at least a separate file for spinning up a basic cluster, where each `provider.sh` script will call it and do extra configuration in its context.
I think it will simplify the provider's script by separating responsibilities (spinning up the cluster vs configuring it) and no need to act conditionally upon cluster kind (if SRIOV... do).

Reply:
We already have kind-x which is maintained by @zhlhahaha for ARM
but it is dedicated to ARM needs iiuc (in the past we discussed if we can combine them, and it was concluded it is better not, because this way it is decoupled)

imho, having this flag here won't hurt, sometimes you need just a cluster of kind, without SR-IOV features
and this simple method allows it

Reply:
We already have kind-x which is maintained by @zhlhahaha for ARM
but it is dedicated to ARM needs iiuc (in the past we discussed if we can combine them, and it was concluded it is better not, because this way it is decoupled)

imho, having this flag here won't hurt, sometimes you need just a cluster of kind, without SR-IOV features
and this simple method allows it

another benefit is that it fixes https://github.com/kubevirt/kubevirtci/issues/991